modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="understanding-clusterversion-conditiontypes_{context}"]
+= Understanding cluster version condition types
+ 
+The Cluster Version Operator (CVO) is responsible for collecting the status conditions of the cluster version and its Operators. This status includes the condition type, informing you of the health and current state of the {product-title} cluster.
+
+In addition to `Available`, `Progressing`, and `Upgradeable`, there are additional condition types that affect cluster versions and Operators.
+
+* Failing: 
+The CVO monitors cluster Operators and other components. `Failing` is set on the cluster version status when a cluster cannot reach the desired state. This indicates that the cluster is not healthy, and requires an administrator to intervene.
+
+* Invalid:
+If the cluster version has an error that prevents the server from taking action, the condition type of the cluster version reports as `Invalid`. The CVO only reconciles the current state as long as this condition is set.
+
+* RetrievedUpdates:
+The cluster version condition `RetrievedUpdates` indicates whether or not available updates have been retrieved from the upstream update server. The condition is `Unknown` before retrieval, `False` if the updates either recently failed or could not be retrieved, or `True` if the `availableUpdates` field is both recent and accurate.
+
+* ReleaseAccepted: 
+The cluster version condition `ReleaseAccepted` has a `True` status if the requested release payload was successfully loaded without failure during image verification and precondition checking.
+
+* ImplicitlyEnabledCapabilities:
+The cluster version condition `ImplicitlyEnabledCapabilities` has a `True` status if there are enabled capabilities that the user is not currently requesting through `spec.capabilities`. The CVO does not support disabling capabilities if any associated resources were previously managed by the CVO.
+
```

Comment:
I am a stickler about parallelism in lists. I would prefer if each entry is in the same format, such as _The cluster version condition `RetrievedUpdates` indicates_..